examples/apps/colorapp/src/colorteller/Dockerfile

```diff
@@ -1,13 +1,15 @@
 FROM golang:1-stretch as builder
+ARG GO_PROXY=direct
```

Comment:
is this what we decided? override the proxy by default? I was suggesting that we dont change the default behavior by default.

Reply:
A user of the example also still can't override this right? We didn't expose this build arg anywhere in the `deploy.sh` script.

Reply:
As per offline discussion, modified to expose this variable.